### ISSUE TITLE
Invalid JSON format treated as valid

### ISSUE THREAD

**Issue by mihaipogonaru** [json-parser/json-parser#73]
The parser doesn't check for the case when there is a ',' followed by nothing so strings like "[1,]" , '{"a":1,}' pass as valid JSONs

**Comment by LB--** [comment]
I believe this is intentional, though if it's really a problem perhaps a parser flag could be added for it?

**Comment by DimitriPapadopoulos** [comment]
Indeed, this is not a JSON validator. It's a JSON parser. Parsers tend to be more lenient. Why is this a problem?
